### PR TITLE
Premium services

### PR DESCRIPTION
**Pull request by queenieho**

### CODE REVIEW

**Comment by allenjprice** [comment]
LTGM 👍

**Comment by jalehman** [comment]
@queenieho In the future, let's try and get better about naming the PR after what it adds to the codebase. If that's too long for a title, then a summary in the description would be good. I haven't been the best at this myself, but I'm finding that it helps contextualize what I'm looking at during the review process.

File: src/cljs/iface/components/services.cljs [hunk 1]
```diff
@@ -111,9 +110,10 @@
    [:div.column
     [ant/form-item {:label (:label field)}
      [ant/input-number
-      {:default-value 1
-       :value         (:value field)
-       :min           1
+      {:value         (if (:value field)
```
Comment (jalehman) [inline src/cljs/iface/components/services.cljs]: `(:value field 0)`—you can provide a default value during lookups.

Comment (allenjprice) [reply src/cljs/iface/components/services.cljs]: whoah TIL

**Pull request merged**